### PR TITLE
UI v2: create a new react app, login and register page

### PR DESCRIPTION
**Pull request by uditmishra128**
I am staring to build new UI and for this I create a react app and in it I create login and register page.

### CODE REVIEW

File: my-app/src/components/Login.component.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export default class Login extends Component {
+  render() {
+    return (
+      <form>
+        <h3>Log in</h3>
+
+        <div className="form-group">
+          <label>Email</label>
+          <input
```
[the remark this answers is not in the document]
Comment (uditmishra128) [reply my-app/src/components/Login.component.js:11]: suggest me how can I resolve It

[the remark this answers is not in the document]
Comment (damiencarol) [reply my-app/src/components/Login.component.js:11]: you need to add the "for" attribute of label element I think.
like:
```html
<div class="preference">
    <label for="cheese">Do you like cheese?</label>
    <input type="checkbox" name="cheese" id="cheese">
</div>
```
see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

File: my-app/src/components/Login.component.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            type="password"
+            className="form-control"
+            placeholder="Enter password"
+          />
+        </div>
+
+        <div className="form-group">
+          <div className="custom-control custom-checkbox">
+            <input
```
[the remark this answers is not in the document]
Comment (uditmishra128) [reply my-app/src/components/Login.component.js:29]: Screenshot (174)
Now it is fine?

File: my-app/src/components/Login/Signup.component.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      <div className="login">
+        <Navbars />
+        <div className="outer">
+          <div className="inner">
+            <form>
+              <h3>Register</h3>
+
+              <div className="form-group">
+                <label for="firstname">First name</label>
+                <input
```
[the remark this answers is not in the document]
Comment (uditmishra128) [reply my-app/src/components/Login/Signup.component.js:16]: Which label? I am not able to understand it.
Plz Explain.

File: my-app/src/components/Login/Login.component.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        <section>
+          <Navbars />
+        </section>
+        <div className="outer">
+          <div className="inner">
+            <form>
+              <h3>Log in</h3>
+
+              <div className="form-group">
+                <label>Email</label>
```
Comment (damiencarol) [inline my-app/src/components/Login/Login.component.js:17]:
```suggestion
                <label for="email">Email</label>
```

File: my-app/src/components/Login/Login.component.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                  type="password"
+                  className="form-control"
+                  placeholder="Enter password"
+                  name="password"
+                />
+              </div>
+
+              <div className="form-group">
+                <div className="custom-control custom-checkbox">
+                  <input
```
[the remark this answers is not in the document]
Comment (uditmishra128) [reply my-app/src/components/Login/Login.component.js:38]: which label?

**Comment by uditmishra128** [comment]
plz review the code

File: my-app/src/components/Login.component.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (damiencarol) [reply my-app/src/components/Login.component.js]: yes this change is good. try to push it to make AccessLint happy

File: my-app/src/components/Login/Signup.component.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (damiencarol) [reply my-app/src/components/Login/Signup.component.js:16]: https://www.w3schools.com/tags/tag_label.asp

**Comment by uditmishra128** [comment]
Sure ,I try my best to make  AccessLint happy

File: ui-2/README.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (damiencarol) [inline ui-2/README.md:1]: please update the readme with some text that explain our experiment

File: ui-2/public/index.html [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+   
+    <title>Login</title>
+  </head>
+  <body>
+    <script src="../src/index.js" type="text/javascript"></script>
```
Comment (damiencarol) [inline ui-2/public/index.html:9]: all these "../src/<foo>" reference will need some work. for exemple change it to just "<foo>"

File: ui-2/src/components/Login/Login.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+            <div className="form-group">
+              <label for="email">Email</label>
+              <input
+                label="email"
+                type="email"
+                className="form-control"
+                placeholder="Enter email"
+                name="email"
+              />
```
Comment (damiencarol) [inline ui-2/src/components/Login/Login.js:23]: add "id" attribute to all inputs and fix the label associated.

File: ui-2/src/components/Login/Navbar.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+export default class Navbars extends Component {
+  render() {
+    return (
+      <div>
+        <Navbar collapseOnSelect expand="lg" bg="light">
+          <Nav className="mr-auto">
+            <NavbarBrand>
+              <Link to={"/sign-in"}>
+                <img
+                  src="https://demo.defectdojo.org/static/dojo/img/logo.png"
```
Comment (damiencarol) [inline ui-2/src/components/Login/Navbar.js:16]: create a copy of the image and store it in the ui-2 folder

**Review by damiencarol — CHANGES_REQUESTED** [review]
you need to revert back doc/branching_model.png and fix some HTML ids

File: ui-2/public/index.html (hunk 7 above)
Comment (uditmishra128) [reply ui-2/public/index.html:9]: sir I have to import the script form index.js

File: ui-2/README.md (hunk 6 above)
Comment (uditmishra128) [reply ui-2/README.md:1]: sure

**Comment by ptrovatelli** [comment]
as discussed, this is just an experiment for now. if we keep it, it'll be probably in another repository. adding the "do not merge" label in case someone wonders

**Pull request closed without merging**